scripts/deploy.cketh.sh

```diff
@@ -30,28 +30,7 @@ dfx deploy cketh_minter --specified-id jzenf-aiaaa-aaaar-qaa7q-cai --network "$D
 })"
 
 echo "Step 3: deploy ledger canister..."
-PRINCIPAL="$(dfx identity get-principal)"
-dfx deploy cketh_ledger --specified-id apia6-jaaaa-aaaar-qabma-cai --network "$DFX_NETWORK" --argument "(variant {
```

Comment:
--specified-id is the same as that provided in dfx.json

--argument is the same as that created by the build command